keanu-project/src/test/java/io/improbable/keanu/tensor/bool/SimpleBooleanTensorTest.java

```diff
@@ -151,4 +151,11 @@ public void canConvertToIntegers() {
         assertArrayEquals(new int[]{1, 0, 1, 0}, matrixA.asFlatIntegerArray());
     }
 
+    @Test
+    public void canReshape() {
```

Comment:
Nice! Not sure how Boolean didn't have a reshape...

Reply:
poor guy was feeling left out